##  -*- python -*-  
# db file automatically generated by genconf on: Thu Sep  2 16:13:06 2010
## insulates outside world against anything bad that could happen
## also prevents global scope pollution
def _fillCfgDb():
    from GaudiKernel.Proxy.ConfigurableDb import CfgDb

    # get a handle on the repository of Configurables
    cfgDb = CfgDb()

    # populate the repository with informations on Configurables 

    cfgDb.add( configurable = 'HltRateMonitor',
               package = 'HltMonitors',
               module  = 'HltMonitors.HltMonitorsConf',
               lib     = 'HltMonitors' )
    cfgDb.add( configurable = 'HltMonitorBase',
               package = 'HltMonitors',
               module  = 'HltMonitors.HltMonitorsConf',
               lib     = 'HltMonitors' )
    cfgDb.add( configurable = 'HltCompositionMonitor',
               package = 'HltMonitors',
               module  = 'HltMonitors.HltMonitorsConf',
               lib     = 'HltMonitors' )
    return #_fillCfgDb
# fill cfgDb at module import...
try:
    _fillCfgDb()
    #house cleaning...
    del _fillCfgDb
except Exception,err:
    print "Py:ConfigurableDb   ERROR Problem with [%s] content!" % __name__
    print "Py:ConfigurableDb   ERROR",err
    print "Py:ConfigurableDb   ERROR   ==> culprit is package [HltMonitors] !"
